app/src/main/java/org/sopt/androidseminar_week2/utils/BlankCheckUtils.kt

```diff
@@ -0,0 +1,15 @@
+package org.sopt.androidseminar_week2.utils
+
+import android.text.Editable
+
+class BlankCheckUtils {
+
+    fun isBlankFilled (list: List<Editable>) : Boolean {
+        for(item in list) {
+            if(item.isBlank()) {
+                return false
+            }
+        }
+        return true
+    }
```

Comment:
- 검증로직을 따로 분리한 이유가 있을까? 왜 클래스로 따로 분리했는 지 이유를 듣고싶어
- 이건 코틀린에서 함수형 API를 사용하면 조금 더 깔끔하게 함수를 만들 수 있을 것 같아 예를 들어서

```suggestion
class BlankCheckUtils {
    fun isBlankFilled (list: List<Editable>) = list.all { it.isNotBlank() } 
}
```

이건 하나의 예지 다른 방식으로도 충분히 구현이 가능해, 다만 코틀린의 특성을 살려서 구현을 해보면 좋을 것 같아

- 이런 검증로직을 "Class"로 만들면 어떤 문제가 발생할 수 있을까? 만약에 어떤 개발자가 한 클래스 내부에서 BlankCheckUtils 객체를 수십개 만들어 놓는다면?(물론 그런일은 거의 없겠지만) 어떤일이 발생할까요?

Reply:
음.. 검증 로직 함수로 만들면서 그냥 아무 생각 없이 분리했던 것 같아..! 반성합니다..
그리고 어떤 문제가 발생할지는 정확히 잘 모르겠어서 다음에 설명 부탁합니다!